### DISCUSSION TITLE
Tabs does not support server-side rendering?

### DISCUSSION THREAD

**Discussion by FlatMapIO** [chakra-ui/ark#2352] [Q&A]
I use ark-ui in solid-start, and I find that the tabs component always active the selected tab one second after the page loads:

https://github.com/chakra-ui/ark/assets/1720349/8bf4419c-5cd5-449a-8de4-cdce6e78e83a

**Comment by cschroeter** [comment]
@FlatMapIO 

Thanks for taking your time to create this issue. The tabs component are rendered on the server.  However the `width` and `height` of the indicator can only be measured on the client side. You may need to use a different styling approach.
